libs/messaging/messaging/src/main/kotlin/net/corda/messaging/api/mediator/RoutingDestination.kt

```diff
@@ -6,9 +6,14 @@ package net.corda.messaging.api.mediator
 data class RoutingDestination(
     val client: MessagingClient,
     val endpoint: String,
+    val type: Type
```

Comment:
```suggestion
    val type: Type = SYNCHRONOUS
```
This will reduce the footprint of the files modified.

Reply:
This is a POC still, not production ready code.